### PR TITLE
Update directory.json

### PR DESCRIPTION
**Pull request by neerajp99**
Sri Lanka Open Source Organisation

### CODE REVIEW

**Comment by roonyh** [comment]
You need a trailing comma to make it valid json.  

More importantly Ceylon Linux is not an open source organization. They are a private business which sells products for open source systems. I know this because I am from Sri Lanka and familiar with it :) 

We are looking for communities that supports and promotes open source. Or open source projects.

**Comment by neerajp99** [comment]
What do you mean by a trailing comma?

File: directory.json [hunk 1]
```diff
@@ -160,4 +160,6 @@
 	"X-LUG": "https://raw.githubusercontent.com/manastaneja/communities/master/Xiyou%20lug",
 	"YGNU": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Yemen/Linux%20Association",
 	"Yoko-lug" : "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Japan/YokohamaLinuxUserGroup-api.json"
```
Comment (roonyh) [inline directory.json]: there should be a comma at the end of this line

**Comment by neerajp99** [comment]
Except this the procedure is correct.
I mean the way I did the task . ?

**Comment by jig08** [comment]
@neerajp99 Working on this ?
Closing this for now...

**Pull request closed without merging**